pkg/network/additional_networks.go

```diff
@@ -57,3 +58,65 @@ func validateRaw(conf *operv1.AdditionalNetworkDefinition) []error {
 
 	return out
 }
+
+// renderOpenShiftSRIOVNetworkCRD returns the manifests of OpenShiftSRIOV NetworkAttachmentDefinition.
+func renderOpenShiftSRIOVNetworkCRD(manifestDir string) ([]*uns.Unstructured, error) {
```

Comment:
can you just combine these in to a single function `renderOpenShiftSRIOV`?

Reply:
Done